scripts/splitControl.js

```diff
@@ -0,0 +1,132 @@
+/*global $ */
+/* exported initSplit */
+
+function initSplit(container, reDraw, config) {
```

Comment:
`splitControl` would perhaps be a better name for this control as reflected in the filename.

Please move the parts of the comment within the function outside of it and make it clearer what the arguments are and what the function returns, eg:
```js
/*
Create a splitter between two <div>s within a container.
Arguments:
* container - ID of a <div> which contains two <div>s (herein referred
  to as pane1 and pane2). The splitter will be created between these.
* reDraw - jQuery callback which is fired when the container sizes change.
* config - dictionary that controls the div:
  {
    splitDirection: "horizontal" or "vertical",
    splitPercent: 50, // what is this?
    minSizePane1: 50,  // describe this
    minSizePane2: 50, // describe this
  }
Returns:
???
*/
function splitControl(container, reDraw, config) {
```

Reply:
I'll do that.